tests/foreman/cli/test_reporttemplates.py

```diff
@@ -857,6 +857,40 @@ def test_positive_schedule_entitlements_report(
     assert local_subscription['name'] in data_csv
 
 
+@pytest.mark.tier3
+def test_entitlements_report_no_inputs_field(
+    module_entitlement_manifest_org, local_ak, local_subscription, rhel7_contenthost, target_sat
+):
+    """Generate an report using the Subscription - Entitlement Report template
+    without passing in the 'Days from Now' argument in the inputs field, to test the
+    default setting
+
+    :id: 5c4e52b9-314c-470d-9946-3d6e05c85b7e
+
+    :steps:
+        1. hammer report-template generate --organization '' --id '' --report-format ''
+
+    :expectedresults: report is generated, and the Days From Now field isn't required
+
+    :BZ: 1943306
+
+    :customerscenario: true
+    """
+    client = rhel7_contenthost
+    client.install_katello_ca(target_sat)
+    client.register_contenthost(module_entitlement_manifest_org.label, local_ak['name'])
```

Comment:
```suggestion
    client.register(
        module_entitlement_manifest_org, module_location, local_ak['name'], target_sat
    )
```
Via global registration.